plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -432,6 +432,7 @@ export class KubernetesBuilder {
             name: cd.name,
             title: cd.title,
             dashboardUrl: cd.dashboardUrl,
+            dashboardParameters: cd.dashboardParameters,
```

Comment:
I noticed that both `dashboardUrl` & `dashboardParameters` do not have error handling for use cases where they might be null, undefined, or even wrongly formatted. While this might be okay if we’re confident **(I have been burned and have trust issues, 🤣 )**, these values are always correctly defined; adding some validation or logging could help 💪🏽  the feature. For example, logging a ⚠️  when missing or incomplete could be helpful for troubleshooting and preventing potential 🔥 . This small addition could go a long way in improving the overall reliability IMHO.

This also is a non-blocking comment, and maintainers can totally ignore it.